synapse/tests/test_cortex.py

```diff
@@ -5968,6 +5968,9 @@ async def test_cortex_storm_lib_dmon_cmds(self):
             msgs = await core.stormlist('dmon.list')
             self.stormIsInPrint('(wootdmon            ): error', msgs)
 
+            # invalid storm query
+            await self.asyncraises(s_exc.BadSyntax, core.nodes('$lib.dmon.add(" | | | ")'))
```

Comment:
if we use self.raises() over self.asyncraises the context manager is also inspectable

Reply:
I didn't see a need to inspect the context manager for this part of the test